demo/test/pickplace_demo.test

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<launch>
+    <include file="$(find moveit_resources_panda_moveit_config)/launch/demo.launch">
+        <arg name="use_rviz" value="false"/>
+        <!-- Enable for speedup once the argument is available upstream
+        <arg name="execution_type" value="last point" />
+        -->
```

Comment:
Enabling this argument requires https://github.com/ros-planning/moveit_resources/pull/61 .
The speedup on my laptop is (surprisingly only) 18 -> 13 seconds.